src/main/kotlin/app/revanced/patches/duolingo/unlocksuper/fingerprints/DuolingoSuperFingerprint.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.duolingo.unlocksuper.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import com.android.tools.smali.dexlib2.AccessFlags
+
+object DuolingoSuperFingerprint : MethodFingerprint(
+    returnType = "V",
+    parameters = "L,L,L,L,L,L,Z,L,L,J,L,L,L,Z,Z,Z,Z,Z,Z,Z,Z,L,L,L,L,L,L,Z,Z,Z,Z,L,L,L,J,I,L,L,L,L,L,L,L,L,L,L,L,L,Z,Z,Z,Z,Z,Z,Z,Z,Z,Z,Z,L,Z,L,L,L,Z,Z,Z,L,L,L,J,L,L,L,L,Z,L,Z,Z,L"
+        .split(","),
```

Comment:
This may easily break in the future. Considering you have listed strings, it is unlikely this fingerprint will match any other method than it does now.